### PR TITLE
PUBDEV-8241 if and eif documentation [nocheck]

### PR DESCRIPTION
**Pull request by valenad1**
https://h2oai.atlassian.net/browse/PUBDEV-8241

I added a part about anomaly score in out anomaly detection algorithms

### CODE REVIEW

File: h2o-docs/src/product/data-science/if.rst [hunk 1]
```diff
@@ -117,6 +117,42 @@ Defining an Isolation Forest Model
...
+
+    - ``1`` = Anomaly
+    - ``0`` = Normal point
+
+Point is marked as an anomaly if the score is great or equal to (1-``contamination``)% quantile of the score.
+
+.. math::
+    predict = score >= Q_{score}(1-contamination)
+
+-  **score**: Normalized mean_length
```
Comment (maurever) [inline h2o-docs/src/product/data-science/if.rst:137]: Why do you use mean_length and then camelcase in maxPathLength and minPathLength? It looks strange.

```suggestion
-  **score**: Normalized meanLength
```

**Review by hannah-tillman — APPROVED** [review]
made a few syntax changes :)

File: h2o-docs/src/product/data-science/if.rst (hunk 1 above)
Comment (valenad1) [reply h2o-docs/src/product/data-science/if.rst]: I have changed it to snake case everywhere according to out conversation.

**Review by maurever — APPROVED** [review]
LGTM, thanks @valenad1!

**Pull request merged**